src/System.Management.Automation/utils/Telemetry.cs

```diff
@@ -112,17 +112,247 @@ static ApplicationInsightsTelemetry()
+                        "Microsoft.Crm.PowerShell",
+                        "Microsoft.DiagnosticDataViewer",
+                        "Microsoft.DirectoryServices.MetadirectoryServices.Config",
+                        "Microsoft.Dynamics.Nav.Apps.Management",
+                        "Microsoft.Dynamics.Nav.Apps.Tools",
+                        "Microsoft.Dynamics.Nav.Ide",
+                        "Microsoft.Dynamics.Nav.Management",
+                        "Microsoft.Dynamics.Nav.Model.Tools",
+                        "Microsoft.Dynamics.Nav.Model.Tools.Crm",
+                        "Microsoft.EnterpriseManagement.Warehouse.Cmdlets",
+                        "Microsoft.Medv.Administration.Commands.WorkspacePackager",
+                        "Microsoft.PowerApps.Checker.PowerShell",
                         "Microsoft.PowerShell.Archive",
+                        "Microsoft.PowerShell.Core",
+                        "Microsoft.PowerShell.Diagnostics",
                         "Microsoft.PowerShell.Host",
+                        "Microsoft.PowerShell.LocalAccounts",
                         "Microsoft.PowerShell.Management",
+                        "Microsoft.PowerShell.ODataUtils",
+                        "Microsoft.PowerShell.Operation.Validation",
                         "Microsoft.PowerShell.Security",
                         "Microsoft.PowerShell.Utility",
+                        "Microsoft.SharePoint.Powershell",
+                        "Microsoft.SystemCenter.ServiceManagementAutomation",
+                        "Microsoft.Windows.ServerManager.Migration",
+                        "Microsoft.WSMan.Management",
+                        "Microsoft.WsMan.Management",
+                        "Microsoft.Xrm.OnlineManagementAPI",
+                        "Microsoft.Xrm.Tooling.CrmConnector.PowerShell",
+                        "Microsoft.Xrm.Tooling.PackageDeployment.Powershell",
+                        "Microsoft.Xrm.Tooling.Testing",
+                        "MicrosoftPowerBIMgmt",
+                        "MicrosoftStaffHub",
+                        "MicrosoftTeams",
+                        "MicrosoftTeams",
+                        "MicrosoftTeams",
+                        "mlSqlPs",
+                        "Mprise.Indigo.Nav.IACop",
+                        "MSOnline",
+                        "MSTeams",
+                        "MSWindowsDefender",
+                        "NickDarvey.ServiceFabric.ConnectionStrings",
+                        "NonPSJob",
+                        "nPSDesiredStateConfiguration",
+                        "nPSDesiredStateConfiguration",
+                        "OperationsManager",
                         "PackageManagement",
+                        "PartnerCenter",
                         "Pester",
+                        "PKITools",
+                        "PoShGroupPolicy",
+                        "PoshPctBar",
+                        "PoshWSUS",
```

Comment:
This is not a Msft module and I suspect others in this are also not.  Please rereview.

Reply:
You are correct, re-did the list and manually verified that all the remaining modules are MSFT owned